src/03-ticket-calculator.js

```diff
@@ -54,7 +54,26 @@ const exampleTicketData = require("../data/tickets");
     calculateTicketPrice(tickets, ticketInfo);
     //> "Entrant type 'kid' cannot be found."
  */
-function calculateTicketPrice(ticketData, ticketInfo) {}
+function calculateTicketPrice(ticketData, ticketInfo) {
+    let ticketPrice =0
+    //error handling
+    if (!ticketData.hasOwnProperty(ticketInfo.ticketType)){
```

Comment:
another way to check if the property exist and therefore is an appropriate ticketType is by checking if `ticketData[ticketInfo.ticketType` and if it falsey or `undefined` then it is safe to say that the ticketType is incorrect.